### PR TITLE
Clean up TestQuerySpillLimits

### LINKED ISSUE #8798: Fix unused properties in TestQuerySpillLimits
After 70a1b37f901e58d752afad5156169a45e6767c15, the `properties` map is no longer used. So either it's not testing what it should, or the map is intentionally no longer used and should be removed.

### PR DESCRIPTION
**Pull request by findepi**
Fixes #8798

### CODE REVIEW

**Comment by kokosing** [comment]
Please see the travis failures.

File: presto-tests/src/test/java/com/facebook/presto/tests/TestQuerySpillLimits.java [hunk 1]
```diff
@@ -64,13 +63,6 @@ public void tearDown()
     public void testMaxSpillPerNodeLimit()
             throws Exception
     {
-        Map<String, String> properties = ImmutableMap.<String, String>builder()
```
Comment (losipiuk) [inline presto-tests/src/test/java/com/facebook/presto/tests/TestQuerySpillLimits.java]: remove `//spill constantly` comment above.
Or replace with more appropriate one.

Edit:

Drop spill related configuration from `SESSION` and use approppriate setters on `FeaturesConfig` below as we already do for some parameters.


Edit2:

Do not set `AGGREGATION_OPERATOR_UNSPILL_MEMORY_LIMIT` it is unrelated to this test.

**Comment by findepi** [comment]
@losipiuk applied

**Comment by losipiuk** [comment]
Merged, thanks!

**Pull request merged**